assets/sass/atoms/_collapse.scss

```diff
@@ -0,0 +1,30 @@
+@charset 'UTF-8';
+
+.collapse-grp {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.collapse-chevron {
+  display: inline-block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -60%) rotate(180deg);
+  font-size: 0.8rem;
+}
+
+.collapse-toggle {
```

Comment:
Not so good looking on screen size around 840px
![screen shot 2017-08-16 at 17 10 53](https://user-images.githubusercontent.com/1708450/29370572-0f238d1a-82a6-11e7-9414-57e472614827.png)